### PR TITLE
Fixed technical launch-related issues

### PR DESCRIPTION
**Pull request by dansagunov**

### CODE REVIEW

**Comment by dekarti** [comment]
i think it's worth squashing into single commit. anyway, it's up to you

**Comment by dansagunov** [comment]
Fair enough. Just wanted everything to be clear for anyone.

**Comment by wlad031** [comment]
We can use "Squash and merge" instead of "Merge pull request". It will allow to us do not use manual squashing

File: app/views.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from flask.ext.httpauth import HTTPBasicAuth
+from flask_httpauth import HTTPBasicAuth
```
Comment (wlad031) [inline app/views.py:1]: Is there any difference?

**Review by wlad031 — APPROVED** [review]

**Comment by dekarti** [comment]
@wlad031 it will be merge commit anyway. Need to decide either we go with 'rebase way' or merge one.

**Comment by wlad031** [comment]
@dekarti no, it is actually "rebase", just Github UI designers don't know this fact)

**Comment by dekarti** [comment]
I thought that "squash and merge" is equivalent to "git merge --squash".

**Comment by wlad031** [comment]
As I understand, it's equivalent to squashing local branch, force pushing it, and normal merging PR.

File: app/views.py (hunk 1 above)
Comment (dansagunov) [reply app/views.py:1]: Yes, there is. Python 3 warns firstly that it's deprecated, then it gives an error.

**Comment by dansagunov** [comment]
By the way, if you follow 'Files changed' tab on top side, you'll find a way to review all commits at once.

**Comment by dekarti** [comment]
@wlad031 it turns out that `git merge --squash` adds all changes to the index without making commit eventually.

**Pull request merged**